<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" ignored="0" total="10" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="143473" started-at="2019-08-05T20:53:47Z" finished-at="2019-08-05T20:56:11Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="143473" started-at="2019-08-05T20:53:47Z" finished-at="2019-08-05T20:56:11Z">
      <class name="com.lojack.qa.testcases.DataDrivenLoginTestCase">
        <test-method status="SKIP" signature="validateCredentials(java.lang.String, java.lang.String)[pri:1, instance:com.lojack.qa.testcases.DataDrivenLoginTestCase@77b52d12]" name="validateCredentials" duration-ms="0" started-at="2019-08-05T20:54:17Z" data-provider="getTestData" finished-at="2019-08-05T20:54:17Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.lang.NullPointerException
at com.lojack.qa.util.LoginData.getTestData(LoginData.java:48)
at com.lojack.qa.testcases.DataDrivenLoginTestCase.getTestData(DataDrivenLoginTestCase.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
... 32 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCredentials -->
      </class> <!-- com.lojack.qa.testcases.DataDrivenLoginTestCase -->
      <class name="com.lojack.qa.testcases.LoginTestCase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="16829" started-at="2019-08-05T20:54:17Z" finished-at="2019-08-05T20:54:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginPageTitleTest()[pri:1, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="loginPageTitleTest" duration-ms="6" started-at="2019-08-05T20:54:34Z" finished-at="2019-08-05T20:54:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="602" started-at="2019-08-05T20:54:34Z" finished-at="2019-08-05T20:54:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="12369" started-at="2019-08-05T20:54:35Z" finished-at="2019-08-05T20:54:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginPagelogoTest()[pri:2, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="loginPagelogoTest" duration-ms="30" started-at="2019-08-05T20:54:47Z" finished-at="2019-08-05T20:54:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPagelogoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="601" started-at="2019-08-05T20:54:47Z" finished-at="2019-08-05T20:54:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="12981" started-at="2019-08-05T20:54:48Z" finished-at="2019-08-05T20:55:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="blankLoginTest()[pri:3, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="blankLoginTest" duration-ms="164" started-at="2019-08-05T20:55:01Z" finished-at="2019-08-05T20:55:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="670" started-at="2019-08-05T20:55:01Z" finished-at="2019-08-05T20:55:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="12034" started-at="2019-08-05T20:55:01Z" finished-at="2019-08-05T20:55:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="incorrectUserLoginTest()[pri:4, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="incorrectUserLoginTest" duration-ms="674" started-at="2019-08-05T20:55:13Z" finished-at="2019-08-05T20:55:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- incorrectUserLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="603" started-at="2019-08-05T20:55:14Z" finished-at="2019-08-05T20:55:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="13508" started-at="2019-08-05T20:55:15Z" finished-at="2019-08-05T20:55:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="incorrectPasswordLoginTest()[pri:5, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="incorrectPasswordLoginTest" duration-ms="606" started-at="2019-08-05T20:55:28Z" finished-at="2019-08-05T20:55:29Z">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<...Please check user...> but was:<... Invalid User ...>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<...Please check user...> but was:<... Invalid User ...>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at com.lojack.qa.testcases.LoginTestCase.incorrectPasswordLoginTest(LoginTestCase.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- incorrectPasswordLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="770" started-at="2019-08-05T20:55:29Z" finished-at="2019-08-05T20:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="12053" started-at="2019-08-05T20:55:30Z" finished-at="2019-08-05T20:55:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="blankUserLoginTest()[pri:6, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="blankUserLoginTest" duration-ms="241" started-at="2019-08-05T20:55:42Z" finished-at="2019-08-05T20:55:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankUserLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="599" started-at="2019-08-05T20:55:42Z" finished-at="2019-08-05T20:55:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="12030" started-at="2019-08-05T20:55:43Z" finished-at="2019-08-05T20:55:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="blankPasswordLoginTest()[pri:7, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="blankPasswordLoginTest" duration-ms="257" started-at="2019-08-05T20:55:55Z" finished-at="2019-08-05T20:55:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankPasswordLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="813" started-at="2019-08-05T20:55:55Z" finished-at="2019-08-05T20:55:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="setUp" is-config="true" duration-ms="13724" started-at="2019-08-05T20:55:56Z" finished-at="2019-08-05T20:56:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="correctcredentialsLoginTest()[pri:8, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="correctcredentialsLoginTest" duration-ms="585" started-at="2019-08-05T20:56:09Z" finished-at="2019-08-05T20:56:10Z">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<...> but was:<...User/Dashboard.aspx>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<...> but was:<...User/Dashboard.aspx>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at com.lojack.qa.testcases.LoginTestCase.correctcredentialsLoginTest(LoginTestCase.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctcredentialsLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.lojack.qa.testcases.LoginTestCase@2d554825]" name="tearDown" is-config="true" duration-ms="595" started-at="2019-08-05T20:56:10Z" finished-at="2019-08-05T20:56:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.lojack.qa.testcases.LoginTestCase -->
      <class name="com.lojack.qa.testcases.ChangePasswordTestCase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.lojack.qa.testcases.ChangePasswordTestCase@614c5515]" name="setUp" is-config="true" duration-ms="19132" started-at="2019-08-05T20:53:47Z" finished-at="2019-08-05T20:54:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="passwordMatchTest()[pri:0, instance:com.lojack.qa.testcases.ChangePasswordTestCase@614c5515]" name="passwordMatchTest" duration-ms="10116" started-at="2019-08-05T20:54:06Z" finished-at="2019-08-05T20:54:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passwordMatchTest -->
      </class> <!-- com.lojack.qa.testcases.ChangePasswordTestCase -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
